internal/keycloakb/configdbmodule.go

```diff
@@ -89,6 +101,11 @@ func NewConfigurationDBModule(db sqltypes.CloudtrustDB, logger log.Logger, actio
 	}
 }
 
+func (c *configurationDBModule) execNoResult(query string, args ...any) error {
```

Comment:
I do not really see the benefit of this function. For me, this kind of wrapping, do not bring something to the code

Reply:
Not a big benefit... just easier to be able to directly return an error rather than affecting it to a variable while ignoring the first returned value then returning the error